ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/SigningEthSendTransactionIntegrationTest.java

```diff
@@ -134,16 +127,21 @@ public void invalidParamsResponseWhenMissingSenderAddress() {
   }
 
   @Test
-  public void signTransactionWhenReceiverAddressIsEmpty() {
-    sendRequestThenVerifyResponse(
-        request.ethSigner(sendTransaction.withReceiver("")), response.ethSigner(INVALID_PARAMS));
-  }
+  public void signTransactionWhenEmptyReceiverAddress() {
```

Comment:
can we do this test, but where the "to" is "0x" (i.e. a _form_ of empty)

Reply:
done